src/pages/organize/[orgId]/projects/[campId]/areaassignments/[areaAssId]/report.tsx

```diff
@@ -94,216 +111,352 @@ const AreaAssignmentReportPage: PageWithLayout<AreaAssignmentReportProps> = ({
+                  )}
+
+                  {metricBeingDeleted?.definesDone ? (
+                    <Box display="flex" flexDirection="column" gap={1}>
+                      {assignment.metrics
+                        .filter(
+                          (metric) =>
+                            metric.kind == 'boolean' &&
+                            metric.id != metricBeingDeleted?.id
+                        )
+                        .map((metric) => (
+                          <Box
+                            key={metric.question}
+                            alignItems="center"
+                            display="flex"
+                            gap={1}
+                            justifyContent="space-between"
+                            width="100%"
                           >
-                            select
-                          </Button>
-                        </Box>
-                      ))}
-                  </Box>
+                            {metric.question || 'Untitled question'}
+                            <Button
+                              onClick={() => {
+                                if (metricBeingDeleted) {
+                                  const filtered = assignment.metrics.filter(
+                                    (metric) =>
+                                      metric.id != metricBeingDeleted.id
+                                  );
+                                  updateAreaAssignment({
+                                    metrics: [
+                                      ...filtered.slice(
+                                        0,
+                                        filtered.indexOf(metric)
+                                      ),
+                                      {
+                                        ...metric,
+                                        definesDone: true,
+                                      },
+                                      ...filtered.slice(
+                                        filtered.indexOf(metric) + 1
+                                      ),
+                                    ],
+                                  });
+                                }
+                                setAnchorEl(null);
+                                setMetricBeingDeleted(null);
+                              }}
+                              variant="outlined"
+                            >
+                              select
+                            </Button>
+                          </Box>
+                        ))}
+                    </Box>
+                  ) : (
+                    <Box display="flex" justifyContent="end" p={2}>
+                      <Button
+                        onClick={() => {
+                          setMetricBeingDeleted(null), setAnchorEl(null);
+                        }}
+                        sx={{ marginRight: 2 }}
+                      >
+                        CANCEL
```

Comment:
These should not be written in all caps. They will get rendered in all caps by `Button`, but that's a styling decision and should not be handled in the content.

(Also, this needs to be internationalized of course)